### ISSUE TITLE
A11y_AzureDataStudioSQL_Projects_CreateNewProject_Keyboard: Keyboard focus is not visible on "Ok" button.

### ISSUE THREAD

**Issue by anthonydresser** [anthonydresser/testissues#4795]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** -A11yMAS;-A11yTCS;-A11ySev2;-SH_AzureDataStudioSQL_Feb2021;-SQL Azure Data Studio;-Win10;-FTP;-DesktopApp;-WCAG2.1.2;-Keyboard;-WCAG2.4.7;

--- Environment Details:
Application Name: Azure Data Studio
Version: 1.27.0-insider
Windows Version: Windows10

--- Repro Steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Tab till SQL Database Project Home Screen with heading "Projects".
3. Tab till "Create New Project" button or "+" button and hit enter.
4. Fill all the field and tab till "Ok" button and check if keyboard focus is visible on the ok button

--- Actual:
Keyboard focus is not visible on "Ok" button.

--- Expected:
Keyboard focus should be visible on "Ok" button.

--- User Impact:
Low vision users will not know where the keyboard focus goes on ok button or not.
